definitions/npm/mocha_v2.4.x/test_mocha_v2.4.x_hooks.js

```diff
@@ -34,11 +39,14 @@ beforeEach((done : Function) => new Promise((res, rej) => {}));
 after(() => {});
 after((done : Function) => {});
 after(() => new Promise((res, rej) => {}));
+after('some description', () => {});
 
 // $ExpectError number. This type is incompatible with function type.
 after((done : number) => {});
 // $ExpectError can't pass done, and return promise at the same time.
 after((done : Function) => new Promise((res, rej) => {}));
+// $ExpectError number. This type is incompatible with function type.
+after('some description', (done : number) => {});
```

Comment:
Not sure why you have `$ExpectError` on this line? This line seems valid to me...

(Note that this is also why the tests are failing: You've asserted that you're expecting Flow to give an error on line 49 by putting the `$ExpectError` comment above it)